src/main/resources/application.properties

```diff
@@ -1,11 +1,14 @@
 # General
 spring.application.name=api-incident
+spring.datasource.url=${config.datasource.url}
+spring.datasource.username=${config.datasource.username}
+spring.datasource.password=${config.datasource.password}
 # OpenAPI
 openapi.title=Incident
 openapi.name=incident
 openapi.version=@project.version@
 # JPA
-spring.jpa.hibernate.ddl-auto=update
-spring.jpa.hibernate.naming.physical-strategy=org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
-spring.jpa.open-in-view=false
-spring.data.jpa.repositories.bootstrap-mode=default
+spring.jpa.hibernate.ddl-auto=validate
+spring.jpa.database-platform=org.hibernate.dialect.MariaDBDialect
+# Flyway
+spring.flyway.enabled=true
```

Comment:
Isn't this property needed as well (since you are introducing flyway on an existing db?)
baselineOnMigrate=true

Reply:
I was not aware of this. Thank you.